lib/components/primitive-components/PlatedHole.ts

```diff
@@ -28,6 +32,9 @@ export class PlatedHole extends PrimitiveComponent<typeof platedHoleProps> {
     if (props.shape === "oval" || props.shape === "pill") {
       return { width: props.outerWidth, height: props.outerHeight }
     }
+    if (props.shape === "circularHoleWithRectPad") {
```

Comment:
Enum convention is dashes in props, props has an error

![image](https://github.com/user-attachments/assets/ca9464bf-c812-4a90-a2fa-bc56b6aff7e2)

Reply:
I think we should maybe switch to snake case enums in props, @techmannih lets refactor to snake case enums. Dash enums are css convention but i think we can start to move to snake case because its just better in ides and matches circuit json better

Reply:
Make sure to update to snake case in the props repo